playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -215,7 +216,7 @@
       --exclude-groups=system:unauthenticated
       --exclude-users=system:anonymous
       --additive-only=true --confirm -o name
-    when: origin_reconcile_bindings | bool or ent_reconcile_bindings | bool
+    when: (not openshift.common.version_gte_3_7 | bool) and (origin_reconcile_bindings | bool or ent_reconcile_bindings | bool)
```

Comment:
Can you get rid of origin_reconcile_bindings and ent_reconcile_bindings on line 192 and 193 above and then drop the second part of this conditional? The master branch should never have to worry about running on versions < 1.0.6 so that can go.

Feel free to push back, just feels like this conditional is getting cumbersome.

Reply:
done